### PR TITLE
login to MC and SC when login to HCP

### PR DESCRIPTION
**Pull request by wanghaoran1988**
### What type of PR is this?

_(bug/feature/cleanup/documentation)_

### Which Jira/Github issue(s) does this PR fix?

This login to MC and SC when we login to HCP and add oc-mc oc-sc to access MC and SC

### Pre-checks (if applicable)

- [x] Ran unit tests locally
- [ ] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: pkg/cli/session/session.go [hunk 1]
```diff
@@ -312,6 +328,24 @@ set -euo pipefail
 	if err != nil {
 		return err
 	}
+	if e.Options.ManagementClusterID != "" {
+		// This add a command to access management cluster
+		err = e.createBin("oc-mc", "oc --kubeconfig="+filepath.Join(e.Path, e.Options.ManagementClusterID, "config")+" $@")
```
Comment (samanthajayasinghe) [inline pkg/cli/session/session.go:333]: Shall we update session help text to show how to use `oc-mc` and `oc-sc` bins ? 

Ref : https://github.com/openshift/backplane-cli/blob/6411c2f1a7c07b07c538b15b76e423d2d8430760/pkg/cli/session/session.go#L417

**Review by mjlshen — COMMENTED** [review]
Not really feedback on the code, but I don't want to login to the MC/SC when I `ocm backplane login` to an HCP. Is backplane-cli the right place for something like this?

E.g. should an SRE just build like a tmux script to run `ocm backplane session`, `ocm backplane session --manager`, and `ocm backplane session --service` if they want to do this?

**Comment by wanghaoran1988** [comment]
> Not really feedback on the code, but I don't want to login to the MC/SC when I `ocm backplane login` to an HCP. Is backplane-cli the right place for something like this?
> 
`ocm backplane login` will not change, only the session command.
> E.g. should an SRE just build like a tmux script to run `ocm backplane session`, `ocm backplane session --manager`, and `ocm backplane session --service` if they want to do this?
Why a separate script? when you run "session", if it has all the related cluster logged in, why not?

**Comment by mjlshen** [comment]
>>Not really feedback on the code, but I don't want to login to the MC/SC when I ocm backplane login to an HCP. Is backplane-cli the right place for something like this?

>ocm backplane login will not change, only the session command.

ack, good to know

>>E.g. should an SRE just build like a tmux script to run ocm backplane session, ocm backplane session --manager, and ocm backplane session --service if they want to do this?

>Why a separate script? when you run "session", if it has all the related cluster logged in, why not?

This is my personal opinion, but I think it's nice to have commands that do one thing well and let people compose them as they need/want to. For this case specifically, there's also somewhat of a security blast radius where an SRE is logging into three or two clusters at the same time; I think we should only get access to a cluster when we need to, not get access to all three clusters "just in case"

**Comment by wanghaoran1988** [comment]
> > > Not really feedback on the code, but I don't want to login to the MC/SC when I ocm backplane login to an HCP. Is backplane-cli the right place for something like this?
> 
> > ocm backplane login will not change, only the session command.
> 
> ack, good to know
> 
> > > E.g. should an SRE just build like a tmux script to run ocm backplane session, ocm backplane session --manager, and ocm backplane session --service if they want to do this?
> 
> > Why a separate script? when you run "session", if it has all the related cluster logged in, why not?
> 
> This is my personal opinion, but I think it's nice to have commands that do one thing well and let people compose them as they need/want to. For this case specifically, there's also somewhat of a security blast radius where an SRE is logging into three or two clusters at the same time; I think we should only get access to a cluster when we need to, not get access to all three clusters "just in case"

Good point, but the login will do nothing except create a SA for user, and from what I can see now, when we login to HCP cluster, there is always a requirement to login to MC, sometime SC, and the HCP namespace is part of the HCP cluster.

**Comment by bmeng** [comment]
@wanghaoran1988 is this still needed?

**Comment by hectorakemp** [comment]
@wanghaoran1988 can we close this? Is this still relevant?

**Comment by bmeng** [comment]
close this PR as it is stale for more than 5 months

**Pull request closed without merging**